src/main/java/uk/gov/hmcts/divorce/divorcecase/model/FinalOrder.java

```diff
@@ -206,6 +213,18 @@ public enum Granted implements HasLabel {
     )
     private YesOrNo finalOrderReminderSentApplicant2;
 
+    @CCD(
+        label = "Link to scanned d36 form"
+    )
+    private Document scannedD36Form;
```

Comment:
wouldnt this be in documents field? might not be needed.

Reply:
AC4 - and the scanned document should show in the Final Order tab